schema/validator.go

```diff
@@ -33,7 +33,9 @@ type Validator string
 type validateDescendantsFunc func(r io.Reader) error
 
 var mapValidateDescendants = map[Validator]validateDescendantsFunc{
-	MediaTypeManifest: validateManifestDescendants,
+	MediaTypeImageConfig:  validateConfigDescendants,
+	MediaTypeManifest:     validateManifestDescendants,
+	MediaTypeManifestList: validateManifestListDescendants,
```

Comment:
The platform check you're adding here has nothing to do with “descendants”.  Can we get a more generic name for this map?  I'm still partial to calling the map of validators [`Validators`][1] ;).

[1]: https://github.com/opencontainers/image-spec/pull/403/files#diff-4a672c41642ea30ebf88bb87685cf72aR42